scripts/ci/requirements.txt

```diff
@@ -0,0 +1 @@
+atlassian-python-api==3.18.0
```

Comment:
It would be good to make a subfolder within `scripts/ci` e.g.: `scripts/ci/jira` and move both the python script and this file inside it, otherwise it can be hard to understand what this `requirements.txt` is referring to if we happen to add more python scripts to this directory.

Reply:
Done.
Ref. https://github.com/Alfresco/acs-packaging/pull/2450